README.md

```diff
@@ -33,9 +33,13 @@ The system comprises the following components:
 ## Code Prerequisites
 
 In order to build and run the code, you need the following installed:
-  - [Node](https://nodejs.org/) (version `16.13.0`)
-  - [NPM](http://npmjs.org/) (version `8.1.0`)
-  - [Wine](https://www.winehq.org/download), if you would like to generate binaries for Windows.
+  - [Node](https://nodejs.org/en/download/) LTS (version `16.13.0`)
```

Comment:
might be worth calling out the code name `lts/gallium`

Reply:
Done